frontend/utility/AdvancedOutput.cpp

```diff
@@ -200,7 +200,19 @@ void AdvancedOutput::UpdateStreamSettings()
 		int keyint_sec = (int)obs_data_get_int(settings, "keyint_sec");
 		obs_service_apply_encoder_settings(main->GetService(), settings, nullptr);
 		if (!enforceBitrate) {
-			blog(LOG_INFO, "User is ignoring service bitrate limits.");
+			int MaxVBitrate, MaxABitrate;
+			obs_service_get_max_bitrate(main->GetService(), &MaxVBitrate, &MaxABitrate);
+
+			std::string videoBitrateStr = MaxVBitrate > 0 ? std::to_string(MaxVBitrate) : "None";
+			std::string audioBitrateStr = MaxABitrate > 0 ? std::to_string(MaxABitrate) : "None";
```

Comment:
```suggestion
			std::string videoBitrateString = maxVideoBitrate > 0 ? std::to_string(maxVideoBitrate) : "None";
			std::string audioBitrateString = maxAudioBitrate > 0 ? std::to_string(maxAudioBitrate) : "None";
```

See above, there is no good reason to write `Str`, as it's an unnecessary abbreviation. Using `String` itself feels a bit superfluous (as it's an `std::string` already) so something like `videoBitRateLogString` would actually be better, but I'd be fine with just `String`.